API network doc updates (#2061)
postmanlabs/postman-docs: src/pages/docs/postman-for-publishers/api-network/add-api-network.md

  - type: link
    name: "API Network guidelines"
    url:  "/docs/postman-for-publishers/api-network/api-submission-guidelines/"
---

Publishing your documentation to the [API Network](https://explore.postman.com/) connects your API to the Postman community. You can publish docs to the network from any team [collection](/docs/postman/collections/intro-to-collections/)—_for individual collections that are not published via a team, you can instead add to the Postman [templates](/docs/postman-for-publishers/postman-templates/add-templates/)_.

Sharing your docs exposes your API to the millions of developers who use Postman, and lets others within the community learn from your projects. By publishing to the API Network, you share your API through a human-readable reference in the form of your docs—and an executable way to interact with your API endpoints inside Postman, via the embedded Run in Postman button.

Your docs can convey detailed guidance on how to make requests to your API endpoints, as well as the workflows required to integrate with your API in general. This provides the information—and actionable data—developers need to get started using your services.

> > To add your API to the network, you will need a public team profile listing. You can [set your profile up before](#setting-up-your-profile) or during publication.

Reviewer: ```suggestion
> To add your API to the network, you will need a public team profile listing. You can [set your profile up before](#setting-up-your-profile) or we can create one for you during publication and you can customize later. 
```
I just want to make sure we make it clear that creating or setting up a profile is not a necessary part of publishing to API Network.

<img alt="Edit Published Docs" src="https://assets.postman.com/postman-docs/edit-published.jpg" width="400px"/>

Click __Allow Collection Discovery__ to set up your network listing.

![Allow Discovery](https://assets.postman.com/postman-docs/allow-discovery.jpg)

Select __Add to API Network__.

### Providing API detail

Include the __API Name__, __Summary__, and __Description__—add the details potential users will need to know in order to find your API and figure out whether they want to use it. Your description can include markdown, and will automatically be populated from any [description you already have included in your collection](/docs/postman/api-documentation/authoring-your-documentation/).

![API Network Detail](https://assets.postman.com/postman-docs/api-network-detail.jpg)

Select the categories that best describe your API and service. API consumers can choose categories to browse APIs in the network from the left sidebar in the Postman app and at [explore.postman.com](https://explore.postman.com).

![Network Categories](https://assets.postman.com/postman-docs/network-in-app.jpg)

### Setting up your team profile

> When you publish to the API Network, your API will be listed as part of your team profile. If you don't already have a team profile, Postman will automatically generate one when you publish.

Reviewer: ```suggestion
When you publish to the API Network, your API will be listed as part of your team profile. If you don't already have a team profile, Postman will automatically generate one when you publish and you can customize later.
```

<img alt="Publish Profile" src="https://assets.postman.com/postman-docs/publish-profile.jpg" width="450px"/>

> Use the __Public Profile Settings__ link or navigate to your [team profile settings](https://go.postman.co/settings/team/public) to enable and fill out the details consumers will see when your API is published and visible in the network.

Reviewer: ```suggestion
Use the __Public Profile Settings__ link or navigate to your [team profile settings](https://go.postman.co/settings/team/public) to enable or customize the details consumers will see when your API is published and visible in the network.
```

![Enable Profile](https://assets.postman.com/postman-docs/enable-profile.jpg)

Include a team name and description that will appear in your API Network listing. You can customize your profile URL, and add a logo and favicon in your [Settings](https://go.postman.co/settings/team/general).

![Profile Detail](https://assets.postman.com/postman-docs/profile-details.jpg)

## Publish to the network

Once your team profile is set up any collection docs you publish will be associated with it. Click __Publish Collection__.

![Publish Team](https://assets.postman.com/postman-docs/publish-as-team.jpg)

> You will receive a notification and email to confirm that your collection is published.

Reviewer: This line could be moved out of this section.
